packages/nc-gui/components/smartsheet/toolbar/ViewActions.vue

```diff
@@ -138,7 +140,7 @@ useMenuCloseOnEsc(open)
 
             <a-menu-divider />
 
-            <a-sub-menu key="download">
+            <a-sub-menu v-if="hasRole('owner', true)" key="download">
```

Comment:
By doing so, all editors wouldn't be able to download as CSV / XLSX. You should make it configurable to cover both use cases.

Reply:
thank you for your review @wingkwong , will look into it